pybamm/models/submodels/interface/sei/sei_growth.py

```diff
@@ -84,9 +84,15 @@ def get_coupled_variables(self, variables):
         if self.options["SEI"] == "reaction limited":
             # alpha = param.alpha
             C_sei = param.C_sei_reaction
+            T = variables["Negative electrode temperature"]
 
             # need to revise for thermal case
-            j_sei = -(1 / C_sei) * pybamm.exp(-0.5 * (delta_phi - j * L_sei * R_sei))
+            j_sei = (
+                -(1 / C_sei)
+                * pybamm.exp(-0.5 * (delta_phi - j * L_sei * R_sei))
```

Comment:
Speaking out of memory, should a similar factor be in the exponential to account for the temperature in the `R T / F`?

Reply:
Oh yeah, it should only be inside the exponential right?

Reply:
I am not sure about the outside as I don't remember the details of the model, but certainly inside the exponential. Are these from Scott's thesis?

Reply:
The ones outside could be specified by making SEI reaction rate a function of temperature (e.g. arrhenius) I think?

Reply:
Actually I don't know where do the ones outside come from exactly...